consent/manager_test_helpers.go

```diff
@@ -571,120 +590,185 @@ func ManagerTests(m Manager, clientManager client.Manager, fositeManager x.Fosit
 			})
 
 			t.Run("case=ListUserAuthenticatedClientsWithFrontChannelLogout", func(t *testing.T) {
-				for i := 0; i <= 10; i++ {
-					c, h := MockConsentRequest(fmt.Sprintf("LUACWFCL-%d", i), false, 0, false, false, false)
-					if i == 5 {
-						c.Client.FrontChannelLogoutURI = "http://some-url.com/"
-					}
-					c.LoginSessionID = "" // otherwise we had to create the login session as well..
-					c.Subject = "subjectLUACWFCL"
-					clientManager.CreateClient(context.TODO(), c.Client) // Ignore errors that are caused by duplication
+				identity1 := "LUACWFCL"
+				identity2 := "LUBCWFCL"
+				identity1Sessions, err := mockCreateLoginSessions(m, identity1)
+				identity2Sessions, err := mockCreateLoginSessions(m, identity2)
 
-					lc, _ := MockAuthRequest(fmt.Sprintf("LUACWFCL-%d", i), true)
-					lc.Challenge = fmt.Sprintf("fk-login-challenge-LUACWFCL-%d", i)
-					lc.SessionID = ""
-					require.NoError(t, m.CreateLoginRequest(context.TODO(), lc))
+				require.NoError(t, err)
 
-					require.NoError(t, m.CreateConsentRequest(context.TODO(), c))
-					_, err = m.HandleConsentRequest(context.TODO(), c.Challenge, h)
-					require.NoError(t, err)
+				identities := map[string][]LoginSession{
+					identity1: identity1Sessions,
+					identity2: identity2Sessions,
 				}
 
-				clients, err := m.ListUserAuthenticatedClientsWithFrontChannelLogout(context.TODO(), "subjectLUACWFCL")
-				require.NoError(t, err)
+				for identity, sessions := range identities {
+					j, k := 0, 1
+					for (j < 2) && (k >= 0) {
+						c, h := MockConsentRequest(fmt.Sprintf("%s-%d", identity, j), false, 0, false, false, false)
```

Comment:
This needs to be prefixed or we get dupliaces in later tests, see: https://github.com/ory/hydra/pull/1687/files#diff-09be186f7d407a9520f1fdb9316b888aL575

Reply:
On another note, the client id is always `0`, but we want client id 1, 2, 3, 4